### DISCUSSION TITLE
when there is a deploment limited by the resource limitations, it will increase a lot of `OutOfCpu` pods

### DISCUSSION THREAD

**Discussion by wang-xiaowu** [k3s-io/k3s#10609] [Q&A]
when there is a deploment limited by the resource limitations, it will increase a lot of `OutOfCpu` pods
how can i resolve it?

**Comment by brandond** [comment]
Get more resources?

**Reply by wang-xiaowu** [reply]
i hope it can be stop in a number, not increase infinitely
